### ISSUE TITLE
Wagtail Production Resources: missing body content

### ISSUE THREAD

**Issue by Cleop** [LDMW/app#437]
<img src="https://user-images.githubusercontent.com/16775804/27687987-da3e004e-5cd0-11e7-9380-d5f4fdc8801e.png" width=500px />

Currently investigating locating and updating/reapplying this missing content.

Relates #434

**Comment by reddog** [comment]
@Cleop Thanks for ensuring this get addressed.

I'm still not 100% on when new issues should be created - I'm not clear if this is a continuation of #434 or a totally new issue.

When new issues are spun out (as in this case), who should move comments over from previous issues in order to ensure the conversation thread is not lost?  See https://github.com/LDMW/app/issues/434#issuecomment-311952888

Thanks.

**Comment by Cleop** [comment]
@reddog 

I would recommend simply popping a line saying: 'relates #434' on this issue so that people can follow the link should they wish. However, the best way to tell whether an issue should be a new one or a continuation is to look at the title of it. The title of #434 was that the site was 'down' (the production database was not loading), this issue (#437) is addressing missing content from the production database. The production database is now loading (#434), but this is a separate issue (#437).

Hope this helps, happy to run through this process in person when we're next in if anything more needs clarifying 😊

**Comment by reddog** [comment]
Thanks for clarifying @Cleop  As my comment on #434 was before it was un-"in-progress"ed and the new issue #437 opened, I'm wondering if everyone realises this extra work should be done (to move comments over)?

And just to clarify, would be able to reply on my comment so that we're all aware if this is lost data, retrievable data or data someone has to re-input, or "lost" due to further code changes required.

Thanks

**Comment by Cleop** [comment]
@reddog We have the old data and are currently restoring it. At the time of your question we were still diagnosing the issue and I didn't want to misinform you.

It was me who changed the `in-progress` label and created this issue, I did them one immediately after the other because as soon as I realised that the lost body content was independent of getting the production database loading again (which we realised when #434 was complete and therefore no longer in progress), hence the need to create this as a new and separate issue.
